db/schema.rb

```diff
@@ -22,6 +22,8 @@
     t.string   "email"
     t.datetime "created_at",   null: false
     t.datetime "updated_at",   null: false
+    t.boolean  "hiring"
+    t.integer  "hire_count"
   end
 
 end
```

Comment:
I'm glad we have these included now. I think that will be important down the road.